Makefile

```diff
@@ -23,13 +23,11 @@ decrypt-scratch:
 	aws kms decrypt --ciphertext-blob fileb://.env.zip.enc.aws --output text --query Plaintext --region ca-central-1 | base64 --decode > .env.zip &&\
 	unzip -o .env.zip
 
-decrypt-production-orig:
-	@cd env/production &&\
-	aws kms decrypt --ciphertext-blob fileb://.env.enc.aws --output text --query Plaintext --region ca-central-1 | base64 --decode > .env
-
-decrypt-staging-orig:
-	@cd env/staging &&\
-	aws kms decrypt --ciphertext-blob fileb://.env.enc.aws --output text --query Plaintext --region ca-central-1 | base64 --decode > .env
```

Comment:
These `*-orig` make targets are from the way we used to do things (a year or 2 ago?) and we can safely delete.